Jenkinsfile

```diff
@@ -0,0 +1,14 @@
+buildMvn {
+  publishModDescriptor = 'no'
+  publishAPI = 'yes'
```

Comment:
Probably for now it makes sense to set all actions to 'no' and enable once corresponding files are being added